lib/rubocop/cop/lint/empty_conditional_body.rb

```diff
@@ -107,13 +107,41 @@ def correct_other_branches(corrector, node)
           end
         end
 
+        def correct_other_branches?(node)
```

Comment:
How about a method name like?

```suggestion
        def require_other_branches_correction?(node)
```